content/docs/14-troubleshooting/15-palette-upgrade.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Palette Upgrade"
+metaTitle: "Palette Upgrade"
+metaDescription: "Troubleshooting steps for errors encountered with upgrade actions."
+icon: ""
+hideToC: false
+fullWidth: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+
+# Palette Upgrades
+
+We recommend you review the [Release Notes](/release-notes) and the [Upgrade Notes](/enterprise-version/upgrade) before attempting to upgrade Palette. Use this information to address common issues that may occur during an upgrade.
+
+
+
+# Ingress Errors
+
+If you receive the following error message when attempting to upgrade to Palette versions greater than Palette 3.4.X in a Kubernetes environment.
```

Comment:
Incomplete sentence. Did you mean to include the sentence "Use the following steps..." here? If so, you can take the suggestion for line 22 and delete line 30.